mrpt_reactivenav2d/src/mrpt_reactivenav2d_node.cpp

```diff
@@ -122,7 +125,7 @@ class ReactiveNav2DNode
 		 * \return false on any error.
 		 */
 #if MRPT_VERSION>=0x150
-		bool getCurrentPoseAndSpeeds(mrpt::math::TPose2D &curPose, mrpt::math::TTwist2D &curVel)
+		bool getCurrentPoseAndSpeeds(mrpt::math::TPose2D &curPose, mrpt::math::TTwist2D &curVel, mrpt::system::TTimeStamp &timestamp)
 		{
```

Comment:
output param `timestamp` should be set to either: 
1)  `mrpt::system::now()` (and add a `MRPT_TODO("get real timestamp")`, or 
2) to the timestamp of where TF says that was the robot pose.